src/treeSitterUtils.ts

```diff
@@ -3,6 +3,9 @@ import { SyntaxNode } from "web-tree-sitter";
 export const getValueNode = (node: SyntaxNode) =>
   node.childForFieldName("value");
 
+export const getTypeAnnotationNode = (node: SyntaxNode) =>
+  node.childForFieldName("type_annotation");
```

Comment:
So it drove me nuts to figure out, but my understanding is that this won't work. 

`type_annotation` is a [named node](https://tree-sitter.github.io/tree-sitter/using-parsers#named-vs-anonymous-nodes), but it generally isn't assigned a [field name](https://tree-sitter.github.io/tree-sitter/using-parsers#node-field-names). I don't believe there's an API for accessing nodes by their names. 

There are sometimes type fields [[1]](https://github.com/tree-sitter/tree-sitter-typescript/blob/28e757a2f498486931b3cb13a100a1bcc9261456/typescript/src/node-types.json#L2295) unfortunately not most cases.